### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by GreatEmerald**
Fixing issues in R CMD check with regards to documentation.

### CODE REVIEW

File: R/bfastts.R [hunk 1]
```diff
@@ -197,7 +197,7 @@ bfastts <- function(data, dates, type = c("irregular", "16-day", "10-day")) {
   
   #  crop time series and return
   bounds = range(which(!is.na(A)))
-  return(ts(A[bounds[1]:bounds[2]], start=time(A)[bounds[1]], end=time(A)[bounds[2]], freq=freq))
+  return(ts(A[bounds[1]:bounds[2]], start=time(A)[bounds[1]], end=time(A)[bounds[2]], frequency=freq))
```
Comment (GreatEmerald) [inline R/bfastts.R:200]: @appelmar Is there any particular reason why you are doing this, instead of `window(A, start=bounds[1], end=bounds[2])`?

Comment (appelmar) [reply R/bfastts.R]: no, see af125c9

File: inst/REFERENCES.bib [hunk 2]
```diff
@@ -0,0 +1,34 @@
+@article{paulodrylands,
+author = {Bernardino, Paulo N. and De Keersmaecker, Wanda and Fensholt, Rasmus and Verbesselt, Jan and Somers, Ben and Horion, Stéphanie},
+title = {Global-scale characterization of turning points in arid and semi-arid ecosystem functioning},
+journal = {Global Ecology and Biogeography},
+volume = {29},
+number = {7},
+pages = {1230-1245},
```
Comment (appelmar) [inline inst/REFERENCES.bib:7]: 1230--1245  ?

File: R/bfast.R [hunk 3]
```diff
@@ -120,18 +123,18 @@ bfast <- function (Yt, h = 0.15, season = c("dummy", "harmonic", "none"),
     si3 <- sin(2 * pi * tl * w * 3)
     smod <- Wt ~ co + si + co2 + si2 + co3 + si3
     # Start the iterative procedure and for first iteration St=decompose result
-    if (decomp == "stlplus") {
-        St <- stlplus(Yt, t=ti, n.p = f, s.window = "periodic", ...)$data[, "seasonal"]
+    St <- if (decomp == "stlplus") {
+        stlplus::stlplus(Yt, t=ti, n.p = f, s.window = "periodic",   ...)$data[, "seasonal"]
```
Comment (mirt001) [inline R/bfast.R:127]:
```suggestion
        stlplus::stlplus(Yt, t=ti, n.p = f, s.window = "periodic", ...)$data[, "seasonal"]
```
I am not sure if this was intentional, but I suppose one space is enough.

File: R/bfast.R [hunk 4]
```diff
@@ -120,18 +123,18 @@ bfast <- function (Yt, h = 0.15, season = c("dummy", "harmonic", "none"),
...
     } else {
-        St <- stl    (Yt, "periodic")$time.series[, "seasonal"]
+        stl             (Yt,                           "periodic")$time.series[, "seasonal"]
```
Comment (mirt001) [inline R/bfast.R:129]:
```suggestion
        stl(Yt, "periodic")$time.series[, "seasonal"]
```

File: R/bfast.R [hunk 5]
```diff
@@ -120,18 +123,18 @@ bfast <- function (Yt, h = 0.15, season = c("dummy", "harmonic", "none"),
...
     }
   }
   else if (season == "dummy") {
     # Start the iterative procedure and for first iteration St=decompose result
-    if (decomp == "stlplus") {
-        St <-  stlplus(Yt, t=ti, n.p = f, s.window = "periodic", ...)$data[, "seasonal"]
+    St <- if (decomp == "stlplus") {
+        stlplus::stlplus(Yt, t=ti, n.p = f, s.window = "periodic", ...)$data[, "seasonal"]
```
Comment (mirt001) [inline R/bfast.R:135]:
```suggestion
        stlplus::stlplus(Yt, t = ti, n.p = f, s.window = "periodic", ...)$data[, "seasonal"]
```

File: R/bfast01.R [hunk 6]
```diff
@@ -8,14 +8,11 @@
...
-#' order/lag/slag/na.action/stl/sbins.
-#' 
+#' `order`/`lag`/`slag`/`na.action`/`stl`/`sbins`.
 #' 2. A linear model with the given formula is fitted. By default a suitable
 #' formula is guessed based on the preprocessing parameters.
-#' 
 #' 3. The model with 1 breakpoint is estimated as well where the breakpoint is
 #' chosen to minimize the segmented residual sum of squares.
-#' 
 #' 4. A sequence of tests the null hypothesis of zero breaks is performed. Each
```
Comment (mirt001) [inline R/bfast01.R:16]: For me it's difficult to understand what this line means. It feels like there might be a grammatical error, but I don't know exactly what the author wanted. Feel free to ignore it, as it is an issue that is actually not in the scope of this pull request.

File: R/bfast01.R [hunk 7]
```diff
@@ -82,9 +81,7 @@
 #' @author Achim Zeileis, Jan Verbesselt
 #' @seealso \code{\link[bfast]{bfastmonitor}},
 #' \code{\link[strucchangeRcpp]{breakpoints}}
-#' @references de Jong R, Verbesselt J, Zeileis A, Schaepman M (2013).  Shifts
-#' in global vegetation activity trends.  \emph{Remote Sensing}, \bold{5},
-#' 1117--1133.  \url{http://dx.doi.org/10.3390/rs5031117}
+#' @references \insertRef{rogierbfast01}{bfast}
 #' 
 #' Zeileis A (2005). A unified approach to structural change tests based on ML
```
Comment (appelmar) [inline R/bfast01.R:86]: to BibTeX?

File: R/bfastlite.R [hunk 8]
```diff
@@ -28,14 +28,15 @@
...
+#' @author Dainius Masiliunas, Jan Verbesselt
 #' @example examples/bfastlite.r
 #'
 #' @export
 
 bfastlite <- function(data, formula=response ~ trend + harmon, order = 3,
                    lag = NULL, slag = NULL, na.action = na.omit,
                    stl = c("none", "trend", "seasonal", "both"),
-                   decomp=c("stlplus", "stl"), sbins=1, ...)
+                   decomp=c("stl", "stlplus"), sbins=1, ...)
```
Comment (mirt001) [inline R/bfastlite.R:39]:
```suggestion
                   decomp = c("stl", "stlplus"), sbins = 1, ...)
```

File: R/bfastlite.R [hunk 9]
```diff
@@ -28,14 +28,15 @@
 #' \item{breakpoints}{output from \code{\link[strucchangeRcpp]{breakpoints}},
 #' containing information about the estimated breakpoints.}
 #' \item{data_pp}{preprocessed data as output by \code{\link{bfastpp}}.}
+#' @author Dainius Masiliunas, Jan Verbesselt
 #' @example examples/bfastlite.r
 #'
 #' @export
 
 bfastlite <- function(data, formula=response ~ trend + harmon, order = 3,
```
Comment (mirt001) [inline R/bfastlite.R:36]:
```suggestion
bfastlite <- function(data, formula = response ~ trend + harmon, order = 3,
```

File: R/bfastpp.R [hunk 10]
```diff
@@ -56,24 +56,18 @@
...
+#' @references \insertRef{janbfastmonitor}{bfast}
 #' @keywords ts
 #' @example examples/bfastpp.r
 #' 
 #' @export bfastpp
 bfastpp<- function(data, order = 3,
                    lag = NULL, slag = NULL, na.action = na.omit,
                    stl = c("none", "trend", "seasonal", "both"),
-                   decomp=c("stlplus", "stl"), sbins=1)
+                   decomp=c("stl", "stlplus"), sbins=1)
```
Comment (mirt001) [inline R/bfastpp.R:67]:
```suggestion
                   decomp = c("stl", "stlplus"), sbins = 1)
```

**Review by appelmar — APPROVED** [review]
Looks good to me, though I have no experience with `Rdpack`

File: inst/REFERENCES.bib (hunk 2 above)
Comment (mirt001) [reply inst/REFERENCES.bib:7]: it checks out https://onlinelibrary.wiley.com/toc/14668238/2020/29/7

**Review by mirt001 — CHANGES_REQUESTED** [review]
only formatting from my side

**Pull request merged**